provider/kubernetes/kubernetes.go

```diff
@@ -36,17 +36,25 @@ const (
 	traefikDefaultIngressClass = "traefik"
 )
 
+// IngressEndpoint holds the endpoint information for the Kubernetes provider
+type IngressEndpoint struct {
+	IP               string `description:"IP used for Kubernetes Ingress endpoints"`
+	Hostname         string `description:"Hostname used for Kubernetes Ingress endpoints"`
+	PublishedService string `description:"Published Kubernetes Service to copy status from"`
```

Comment:
I don't understand why it's a "published service". Can you explain a bit why you chose the name?

Reply:
I think the name is copied from the Nginx ingress controller, It makes sense to have the same name.

I think the idea is that this is the service that publishes the ingress controller to the outside world.

Reply:
Thanks!